src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -11173,7 +11182,7 @@ private ParameterSyntax ParseLambdaParameter(bool isFirst, ref bool hasTypes)
                     refOrOutOrParams = this.EatToken();
                 }
 
-                paramType = this.ParseType(true);
+                paramType = this.ParseType();
```

Comment:
Though, in that case 'ref' has been consumed by the caller...

---
In reply to: [79703034](https://github.com/dotnet/roslyn/pull/13814#discussion_r79703034) [](ancestors = 79703034,79702440,79675184)